runp tests fail because runp is missing

Found out runp tests are failing because of missing runp while trying to package v1.4.1 in debian.
Logs:
```py
I: pybuild base:310: cd /home/weepingclown/debian/py/plac/python-plac/.pybuild/cpython3_3.12_plac/build; python3.12 -m pytest -vv
==================================================================================================================================================== test session starts ====================================================================================================================================================
platform linux -- Python 3.12.1, pytest-7.4.3, pluggy-1.3.0 -- /usr/bin/python3.12
cachedir: .pytest_cache
rootdir: /home/weepingclown/debian/py/plac/python-plac
collected 28 items                                                                                                                                                                                                                                                                                                          

doc/test_ishelve.py::test PASSED                                                                                                                                                                                                                                                                                      [  3%]
doc/test_ishelve_more.py::test PASSED                                                                                                                                                                                                                                                                                 [  7%]
doc/test_pi.py::test PASSED                                                                                                                                                                                                                                                                                           [ 10%]
doc/test_plac.py::test_expected_help PASSED                                                                                                                                                                                                                                                                           [ 14%]
doc/test_plac.py::test_p1 PASSED                                                                                                                                                                                                                                                                                      [ 17%]
doc/test_plac.py::test_p2 PASSED                                                                                                                                                                                                                                                                                      [ 21%]
doc/test_plac.py::test_p3 PASSED                                                                                                                                                                                                                                                                                      [ 25%]
doc/test_plac.py::test_p4 PASSED                                                                                                                                                                                                                                                                                      [ 28%]
doc/test_plac.py::test_p5 PASSED                                                                                                                                                                                                                                                                                      [ 32%]
doc/test_plac.py::test_global PASSED                                                                                                                                                                                                                                                                                  [ 35%]
doc/test_plac.py::test_flag_with_default PASSED                                                                                                                                                                                                                                                                       [ 39%]
doc/test_plac.py::test_metavar_no_defaults PASSED                                                                                                                                                                                                                                                                     [ 42%]
doc/test_plac.py::test_metavar_with_defaults PASSED                                                                                                                                                                                                                                                                   [ 46%]
doc/test_plac.py::test_metavar_empty_string PASSED                                                                                                                                                                                                                                                                    [ 50%]
doc/test_plac.py::test_kwargs PASSED                                                                                                                                                                                                                                                                                  [ 53%]
doc/test_plac.py::test_kwargs2 PASSED                                                                                                                                                                                                                                                                                 [ 57%]
doc/test_plac.py::test_kwargs3 PASSED                                                                                                                                                                                                                                                                                 [ 60%]
doc/test_plac.py::test_date_default PASSED                                                                                                                                                                                                                                                                            [ 64%]
doc/test_plac.py::test_int_default PASSED                                                                                                                                                                                                                                                                             [ 67%]
doc/test_plac.py::test_none_default PASSED                                                                                                                                                                                                                                                                            [ 71%]
doc/test_plac.py::test_cmds PASSED                                                                                                                                                                                                                                                                                    [ 75%]
doc/test_plac.py::test_cmd_abbrevs PASSED                                                                                                                                                                                                                                                                             [ 78%]
doc/test_plac.py::test_sub_help PASSED                                                                                                                                                                                                                                                                                [ 82%]
doc/test_plac.py::test_yield PASSED                                                                                                                                                                                                                                                                                   [ 85%]
doc/test_plac.py::test_doctest PASSED                                                                                                                                                                                                                                                                                 [ 89%]
doc/test_runp.py::test1 FAILED                                                                                                                                                                                                                                                                                        [ 92%]
doc/test_runp.py::test2 FAILED                                                                                                                                                                                                                                                                                        [ 96%]
doc/test_server.py::test PASSED                                                                                                                                                                                                                                                                                       [100%]

========================================================================================================================================================= FAILURES ==========================================================================================================================================================
___________________________________________________________________________________________________________________________________________________________ test1 ___________________________________________________________________________________________________________________________________________________________

    def test1():
>       assert ['3', '5', '10'] == plac.runp([gen(3), gen(5), gen(10)])
E       AttributeError: module 'plac' has no attribute 'runp'

/home/weepingclown/debian/py/plac/python-plac/.pybuild/cpython3_3.12_plac/build/doc/test_runp.py:17: AttributeError
___________________________________________________________________________________________________________________________________________________________ test2 ___________________________________________________________________________________________________________________________________________________________

    def test2():
>       result, error = plac.runp([gen(3), err()])
E       AttributeError: module 'plac' has no attribute 'runp'

/home/weepingclown/debian/py/plac/python-plac/.pybuild/cpython3_3.12_plac/build/doc/test_runp.py:20: AttributeError
================================================================================================================================================== short test summary info ==================================================================================================================================================
FAILED doc/test_runp.py::test1 - AttributeError: module 'plac' has no attribute 'runp'
FAILED doc/test_runp.py::test2 - AttributeError: module 'plac' has no attribute 'runp'
=============================================================================================================================================== 2 failed, 26 passed in 0.16s ================================================================================================================================================
E: pybuild pybuild:395: test: plugin distutils failed with: exit code=1: cd /home/weepingclown/debian/py/plac/python-plac/.pybuild/cpython3_3.12_plac/build; python3.12 -m pytest -vv
I: pybuild base:310: cd /home/weepingclown/debian/py/plac/python-plac/.pybuild/cpython3_3.11_plac/build; python3.11 -m pytest -vv
==================================================================================================================================================== test session starts ====================================================================================================================================================
platform linux -- Python 3.11.7, pytest-7.4.3, pluggy-1.3.0 -- /usr/bin/python3.11
cachedir: .pytest_cache
rootdir: /home/weepingclown/debian/py/plac/python-plac
collected 28 items                                                                                                                                                                                                                                                                                                          

doc/test_ishelve.py::test PASSED                                                                                                                                                                                                                                                                                      [  3%]
doc/test_ishelve_more.py::test PASSED                                                                                                                                                                                                                                                                                 [  7%]
doc/test_pi.py::test PASSED                                                                                                                                                                                                                                                                                           [ 10%]
doc/test_plac.py::test_expected_help PASSED                                                                                                                                                                                                                                                                           [ 14%]
doc/test_plac.py::test_p1 PASSED                                                                                                                                                                                                                                                                                      [ 17%]
doc/test_plac.py::test_p2 PASSED                                                                                                                                                                                                                                                                                      [ 21%]
doc/test_plac.py::test_p3 PASSED                                                                                                                                                                                                                                                                                      [ 25%]
doc/test_plac.py::test_p4 PASSED                                                                                                                                                                                                                                                                                      [ 28%]
doc/test_plac.py::test_p5 PASSED                                                                                                                                                                                                                                                                                      [ 32%]
doc/test_plac.py::test_global PASSED                                                                                                                                                                                                                                                                                  [ 35%]
doc/test_plac.py::test_flag_with_default PASSED                                                                                                                                                                                                                                                                       [ 39%]
doc/test_plac.py::test_metavar_no_defaults PASSED                                                                                                                                                                                                                                                                     [ 42%]
doc/test_plac.py::test_metavar_with_defaults PASSED                                                                                                                                                                                                                                                                   [ 46%]
doc/test_plac.py::test_metavar_empty_string PASSED                                                                                                                                                                                                                                                                    [ 50%]
doc/test_plac.py::test_kwargs PASSED                                                                                                                                                                                                                                                                                  [ 53%]
doc/test_plac.py::test_kwargs2 PASSED                                                                                                                                                                                                                                                                                 [ 57%]
doc/test_plac.py::test_kwargs3 PASSED                                                                                                                                                                                                                                                                                 [ 60%]
doc/test_plac.py::test_date_default PASSED                                                                                                                                                                                                                                                                            [ 64%]
doc/test_plac.py::test_int_default PASSED                                                                                                                                                                                                                                                                             [ 67%]
doc/test_plac.py::test_none_default PASSED                                                                                                                                                                                                                                                                            [ 71%]
doc/test_plac.py::test_cmds PASSED                                                                                                                                                                                                                                                                                    [ 75%]
doc/test_plac.py::test_cmd_abbrevs PASSED                                                                                                                                                                                                                                                                             [ 78%]
doc/test_plac.py::test_sub_help PASSED                                                                                                                                                                                                                                                                                [ 82%]
doc/test_plac.py::test_yield PASSED                                                                                                                                                                                                                                                                                   [ 85%]
doc/test_plac.py::test_doctest PASSED                                                                                                                                                                                                                                                                                 [ 89%]
doc/test_runp.py::test1 FAILED                                                                                                                                                                                                                                                                                        [ 92%]
doc/test_runp.py::test2 FAILED                                                                                                                                                                                                                                                                                        [ 96%]
doc/test_server.py::test PASSED                                                                                                                                                                                                                                                                                       [100%]

========================================================================================================================================================= FAILURES ==========================================================================================================================================================
___________________________________________________________________________________________________________________________________________________________ test1 ___________________________________________________________________________________________________________________________________________________________

    def test1():
>       assert ['3', '5', '10'] == plac.runp([gen(3), gen(5), gen(10)])
E       AttributeError: module 'plac' has no attribute 'runp'

/home/weepingclown/debian/py/plac/python-plac/.pybuild/cpython3_3.11_plac/build/doc/test_runp.py:17: AttributeError
___________________________________________________________________________________________________________________________________________________________ test2 ___________________________________________________________________________________________________________________________________________________________

    def test2():
>       result, error = plac.runp([gen(3), err()])
E       AttributeError: module 'plac' has no attribute 'runp'

/home/weepingclown/debian/py/plac/python-plac/.pybuild/cpython3_3.11_plac/build/doc/test_runp.py:20: AttributeError
================================================================================================================================================== short test summary info ==================================================================================================================================================
FAILED doc/test_runp.py::test1 - AttributeError: module 'plac' has no attribute 'runp'
FAILED doc/test_runp.py::test2 - AttributeError: module 'plac' has no attribute 'runp'
=============================================================================================================================================== 2 failed, 26 passed in 0.15s ================================================================================================================================================
E: pybuild pybuild:395: test: plugin distutils failed with: exit code=1: cd /home/weepingclown/debian/py/plac/python-plac/.pybuild/cpython3_3.11_plac/build; python3.11 -m pytest -vv
```

--- a/plac.py
+++ b/plac.py
@@ -27,7 +27,7 @@ See docs/index.html for the documentation.
 """
 from plac_core import *
 from plac_ext import (import_main, ReadlineInput, Interpreter,
-                      stdout,  Monitor, default_help)
+                      stdout, runp, Monitor, default_help)
 
 __version__ = '1.4.1'
 

